src/main/java/ilex/util/StdOutLogger.java

```diff
@@ -0,0 +1,32 @@
+package ilex.util;
+
+public class StdOutLogger extends Logger {
+
+    StdOutLogger()
+    {
+        super("");
+    }
+
+    StdOutLogger(String procName)
+    {
+        super(procName);
+    }
+
+    @Override
+    public void close()
+    {
+        // leave System.out alone        
+    }
+
+    /**
+	* Logs a message.
+	* @param priority the priority
+	* @param text the formatted text
+	*/
+    @Override
+    public void doLog(int priority, String text)
+    {
+        System.out.println(standardMessage(priority, text));        
```

Comment:
Why not JUL here? Seems like a good place to start as well as setting up for enterprise logging configurations with cloud deployments.

Reply:
I didn't want to have extra config files involved at this point.